telegraf/init.sls

```diff
@@ -9,6 +9,15 @@ influxdata_repo:
     - file: /etc/apt/sources.list.d/influxdata.list
     - require_in:
       - pkg: telegraf
+{% elif grains['os_family'] == 'RedHat' %}
+influxdata_repo:
+  pkgrepo.managed:
+    - humanname: CentOS-$releasever - Base
```

Comment:
Maybe use the os name grain, instead of fixing it to `CentOS` ?

Reply:
I've changed it to use grains OS